Test/Unit/BoltTestCase.php

```diff
@@ -3,13 +3,318 @@
 namespace Bolt\Boltpay\Test\Unit;
 
 use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+use ReflectionException;
+use ReflectionObject;
+use ReflectionProperty;
+use PHPUnit\Framework\Constraint\IsType;
 
-class BoltTestCase extends TestCase
-{
-    protected function skipTestInUnitTestsFlow()
+if (!defined('PHP_CURRENT_VERSION')) {
+    define('PHP_CURRENT_VERSION', (float)phpversion());
+}
+
+if (PHP_CURRENT_VERSION < 7.2) {
```

Comment:
Magento 2.2.x supports PHP 7.0.x ~ 7.1.x
and the version of PHPUnit is 6.2.4

Magento 2.3.x supports PHP 7.1.3 ~ 7.3.x
and the version of PHPUnit is 6.5.14

Magento 2.4.x supports PHP 7.4.x
and the version of PHPUnit is 9.1.5

In PHPUnit 9, it adds PHP return type declarations to the functions, but PHP 7.0 has limited supports for return type declarations, especially it does not support `void` as return type, the void return type is introduced in PHP 7.1 . Also the method TestCase::createPartialMock has specific return type \PHPUnit\Framework\MockObject\MockObject, as a result, to make our tests fully work with Magento 2.2.x, I create this condition

Reply:
Could you explain again why you change 7.1 to 7.2?

Reply:
I have changed the conditional statement as you suggest.